### PR TITLE
SWATCH-2673: Remove character limit from events instance_id column

### PR DESCRIPTION
**Pull request by kflahert**
Jira issue: SWATCH-2673

## Testing

IQE Test MR: 

### Setup

1. Start app with: `DEV_MODE=true ./gradlew :bootRun`

### Steps

1. Create event using internal api:
```
curl -X 'POST'   'http://localhost:8000/api/rhsm-subscriptions/v1/internal/rpc/tally/events' -H 'Origin: .redhat.com'  -H 'accept: application/json'   -H 'x-rh-swatch-psk: placeholder'   -H 'Content-Type: application/json'   -d '[
  {
    "event_id": "0998c49b-9552-463f-b363-0a4b046dd6e9",
    "event_source": "prometheus",
    "event_type": "snapshot_redhat.com:BASILISK:storage_gb",
    "account_number": "account123",
    "org_id": "org123",
    "service_type": "BASILISK Instance",
    "instance_id": "c5mu16smf1c22rn8e730morethan60characters1000000000000000000000000000000000000000000000000000000000000000000",
    "timestamp": "2021-10-18T22:00:00Z",
    "expiration": "2021-10-19T22:00:00Z",
    "display_name": "c5mu16smf1c22rn8e730",
    "measurements": [
      {
        "value": 1.5,
        "uom": "Storage-gibibyte-months"
      }
    ],
    "role": "BASILISK",
    "sla": "Premium",
    "billing_provider": "red hat",
    "billing_account_id": "placeholderId"
  }
]'

```
Verification:
1. Check database that record is saved: `SELECT * from events where event_id = '0998c49b-9552-463f-b363-0a4b046dd6e9';`

### CODE REVIEW

**Review by Sgitario — APPROVED** [review]
It seems that the both varchar(N) and the text types have the same performance, so everything good here.

File: src/main/resources/liquibase/202406261514-alter-event-instance-id-length.xml
Comment (lindseyburnett) [inline src/main/resources/liquibase/202406261514-alter-event-instance-id-length.xml:1]: Could you add a `<rollback>` instruction to this?

**Review by lindseyburnett — CHANGES_REQUESTED** [review]

**Review by liwalker-rh — APPROVED** [review]
Looks good after testing. Created an IQE MR to test instance ids of different lengths: https://gitlab.cee.redhat.com/insights-qe/iqe-rhsm-subscriptions-plugin/-/merge_requests/768

**Pull request merged**